src/Orleans/Core/Grain.cs

```diff
@@ -308,5 +312,20 @@ protected TGrainState State
         {
             get { return base.GrainState as TGrainState; }
         }
+
+        public Task ClearStateAsync()
```

Comment:
How? I can still see it. Do you want to allow a `Grain` to clear/read/write another `Grain`'s state?